### PR TITLE
Admin: change tags to annotations

### PR DESCRIPTION
**Pull request by begelundmuller**
Changes:

- Changes superuser project "tags" (string array) to "annotations" (string KV-pairs).
- The annotations are propagated to the runtime and set as attributes on *some* telemetry (such as events and request/response logs). 

Example usage:

```bash
rill sudo annotations set org project --annotation foo=bar --annotation hello=world
rill sudo project search --annotation foo=bar
rill sudo project search --annotation foo=*
```

### CODE REVIEW

File: admin/projects.go [hunk 1]
```diff
@@ -133,9 +134,12 @@ func (s *Service) TeardownProject(ctx context.Context, p *database.Project) erro
 func (s *Service) UpdateProject(ctx context.Context, proj *database.Project, opts *database.UpdateProjectOptions) (*database.Project, error) {
 	requiresReset := (proj.Region != opts.Region) || (proj.ProdSlots != opts.ProdSlots)
 
+	log.Printf("HERE: projects=%v AND opts=%v", proj.Annotations, opts.Annotations)
```
Comment (kaspersjo) [inline admin/projects.go:137]: Some left behind debug log?

File: cli/cmd/sudo/annotations/tags.go
Comment (kaspersjo) [inline cli/cmd/sudo/annotations/tags.go:1]: Nitpick but `annotations.go` seems to be a better name?

**Review by kaspersjo — APPROVED** [review]
Two small comments, but otherwise looks good to me!

File: admin/projects.go (hunk 1 above)
Comment (begelundmuller) [reply admin/projects.go]: Whoops 😅

File: cli/cmd/sudo/annotations/tags.go
Comment (begelundmuller) [reply cli/cmd/sudo/annotations/tags.go:1]: Agree!

**Pull request merged**